tests/vmi_multus_test.go

```diff
@@ -554,6 +554,8 @@ var _ = Describe("Multus", func() {
 				Expect(interfaceByIfcName["ep2"].InterfaceName).To(Equal("ep2"))
 				Expect(interfaceByIfcName["ep2"].IP).To(Equal(ep2Ip))
 				Expect(interfaceByIfcName["ep2"].IPs).To(Equal([]string{ep2Ip, ep2IpV6}))
+
+				Expect(updatedVmi.Status.GuestOSInfo.Name).To(Equal("Fedora"))
```

Comment:
I would suggest to make a dedicated test in `tests/vmi_configuration_test.go`. There we already have two agent-specific tests. It should be easy to copy and adapt this test: `VMI condition should signal agent presence`.